src/js/dex/order.service.js

```diff
@@ -0,0 +1,95 @@
+(function () {
+    'use strict';
+
+    function serializeMoney (amount) {
+        return {
+            amount: amount.toTokens(),
+            currency: amount.currency
+        };
+    }
+
+    function deserializeCurrency(json) {
+        // It's not a perfect solution.
+        // Currencies created here won't be equivalent (===) to predefined currencies
+        return new Currency(json.currency);
+    }
+
+    function deserializeMoney (json) {
+        return Money.fromTokens(json.amount, deserializeCurrency(json.currency));
+    }
+
+    function serializeOrder (order) {
+        return {
+            id: order.id,
+            status: order.status,
+            type: order.type,
+            quantity: serializeMoney(order.quantity),
+            price: serializeMoney(order.price),
+            total: serializeMoney(order.total)
+        };
+    }
+
+    function deserializeOrder (json) {
+        return {
+            id: json.id,
+            status: json.status,
+            type: json.type,
+            quantity: deserializeMoney(json.quantity),
```

Comment:
Is it possible to change the field name from `quantity` to `amount`? That's how it's called on back end.